### PR TITLE
feat: RDCC-1695 Authorisation file testing

### PR DESCRIPTION
**Pull request by shreedhar-hmcts**
### JIRA link (if applicable) ###
RDCC-1695 Authorisation file testing

### Change description ###
RDCC-1695 Authorisation file testing

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Library should be included with build.gradle like follows
 compile group: 'uk.gov.hmcts.reform', name: 'data-ingestion-lib', version: '0.2.5'
 And release versions library can be found in bintray (https://bintray.com/hmcts/hmcts-maven/data-ingestion-lib)  
 
+
```
Comment (abhijitdiwan) [inline README.md:17]: revert this file

File: src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java [hunk 2]
```diff
@@ -1,39 +1,201 @@
...
+        judicialOfficeAuthorisation.setCreatedDate(handleNull(columns.get(5), true));
+        judicialOfficeAuthorisation.setLastUpdated(handleNull(columns.get(6), true));
+        judicialOfficeAuthorisation.setLowerLevel(handleNull(columns.get(7), false));
+        return judicialOfficeAuthorisation;
+    }
+
+    static String handleNull(String fieldValue, boolean timeStampField) {
+        if (isBlank(fieldValue) && !timeStampField) {
+            return StringUtils.EMPTY;
+        } else if (timeStampField && isBlank(fieldValue)) {
```
Comment (abhijitdiwan) [inline src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java:185]: https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/ObjectUtils.html#defaultIfNull(T,%20T)

you can use this.

File: src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAuthorisationProcessor.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    @Override
+    public void process(Exchange exchange) {
+
+        List<JudicialOfficeAuthorisation> judicialOfficeAuthorisations;
+
+        judicialOfficeAuthorisations = (exchange.getIn().getBody() instanceof List)
+                ? (List<JudicialOfficeAuthorisation>) exchange.getIn().getBody()
+                : singletonList((JudicialOfficeAuthorisation) exchange.getIn().getBody());
+
+        log.info("Judicial Authorisation Records count before Validation:: " + judicialOfficeAuthorisations.size());
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAuthorisationProcessor.java:43]: remove + :)

File: src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java (hunk 2 above)
Comment (shreedhar-hmcts) [reply src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/ParentIntegrationTestSupport.java:185]: Yes checked the methods, but somehow doesn't fit for the custom condition.
Here for String field i want EMPTY value  and for timestamp field wanted null value, this cant be achieved via apache util method.

File: src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAuthorisationProcessor.java (hunk 3 above)
Comment (shreedhar-hmcts) [reply src/main/java/uk/gov/hmcts/reform/juddata/camel/processor/JudicialOfficeAuthorisationProcessor.java]: done

File: README.md (hunk 1 above)
Comment (shreedhar-hmcts) [reply README.md]: done

**Pull request merged**